rapids-cmake/export/template/build_package_components_versioned.cmake.in

```diff
@@ -0,0 +1,16 @@
+#=============================================================================
+# find_dependency Search for @name@
+#
+# Make sure we search for a build-dir config module for the project
+set(possible_package_dir "@possible_dir@")
+if(possible_package_dir AND NOT DEFINED @name@_DIR)
+  set(@name@_DIR "${possible_package_dir}")
+endif()
+
+find_package(@name@ @version@ QUIET COMPONENTS @components@)
```

Comment:
Is there no way to make this entire line a variable for replacement so that you could combine the versioned and non-versioned files? Same question for the `install_package*` equivalents.

Reply:
This is a good question. I think I can unify everything.
Let me see if it makes things easier to read

Reply:
When I moved to doing this in CMake logic it became harder to see what the final output of the request would be. While the current approach has lots of file the upside is a clearer understanding of what the template looks like

Reply:
OK that's fine to leave as is, then.